src/ng/directive/select.js

```diff
@@ -4,6 +4,16 @@
 
 var noopNgModelController = { $setViewValue: noop, $render: noop };
 
+function setOptionSelectedStatus(optionEl, value) {
+  optionEl.prop('selected', value); // needed for IE
+  /**
+   * When unselecting an option, setting the property to null / false should be enough
+   * However, screenreaders might react to the selected attribute instead, see
+   * https://github.com/angular/angular.js/issues/14419
+   */
+  optionEl.attr('selected', value || null);
```

Comment:
Oh, you little rascal! 
We have the toBeMarkedAsSelected() custom matcher which complains when the selected attr is in the wrong state. And jqlite also removes the attribute when you pass `false` and the element is boolean. So I'm going with that.